src/2023/call-for-speakers/index.html

```diff
@@ -11,8 +11,12 @@
-1.965-7.615-2.62-12.201l-.246-1.474c-1.146-7.697-2.948-10.236-7.943-11.792v-.409c6.387-2.047 9.99-7.37 9.99-14.658 0-10.973-7.124-16.787-20.553-16.787h-25.14v57.321Zm13.594-30.708v-16.131h9.007c6.305 0 9.335 2.538 9.335 8.025 0 5.486-3.03 8.106-9.335 8.106h-9.007ZM215.344 152.651c14.003 0 20.39-6.223 20.39-19.489v-23.256h-13.266v25.631c0 5.077-2.456 7.697-7.124 7.697-4.504 0-7.042-2.702-7.042-7.697v-25.631h-13.266v23.256c0 13.348 6.387 19.489 20.308 19.489ZM269.769 152.651c10.891 0 17.605-8.27 17.605-21.7 0-13.593-6.551-21.618-17.605-21.618-5.323 0-10.154 2.457-12.857 6.551h-.409V93.037h-13.266v58.959h12.693V146.1h.327c2.539 4.013 7.698 6.551 13.512 6.551Zm-13.757-21.7c0-7.615 3.193-11.955 8.762-11.955 5.732 0 9.007 4.34 9.007 11.955 0 7.697-3.275 11.956-9.007 11.956-5.569 0-8.762-4.259-8.762-11.956ZM299.094 167.554h13.265l23.256-57.648h-13.429l-9.827 29.889h-.327l-9.908-29.889h-13.921l17.114 42.09-6.223 15.558Z"/><path fill="#F90101" d="M353.643 151.996h36.734v-9.859h-13.5v-.314l4.019-4.144c6.342-6.468 8.477-10.612 8.477-16.201 0-8.728-6.468-14.003-17.08-14.003-11.115 0-17.896 5.84-18.273 15.824h13.312c.251-4.395 1.696-6.279 4.961-6.279 2.512 0 3.767 1.381 3.767 4.144 0 3.014-1.569 5.715-6.719 11.868l-15.698 18.587v.377ZM413.151 152.561c13.187 0 20.094-7.786 20.094-22.543 0-14.882-6.845-22.543-20.094-22.543-13.25 0-20.031 7.661-20.031 22.543 0 14.757 6.907 22.543 20.031 22.543Zm-6.154-22.543c0-9.168 1.758-12.684 6.154-12.684 4.458 0 6.216 3.516 6.216 12.684s-1.758 12.685-6.216 12.685c-4.396 0-6.154-3.517-6.154-12.685ZM434.774 151.996h36.735v-9.859h-13.501v-.314l4.019-4.144c6.342-6.468 8.477-10.612 8.477-16.201 0-8.728-6.468-14.003-17.08-14.003-11.114 0-17.896 5.84-18.273 15.824h13.312c.252-4.395 1.696-6.279 4.961-6.279 2.512 0 3.768 1.381 3.768 4.144 0 3.014-1.57 5.715-6.719 11.868l-15.699 18.587v.377ZM492.915 152.561c11.429 0 18.085-5.714 18.085-15.447 0-7.221-3.956-11.994-11.114-13.438v-.377L511 108.417v-.377h-35.792v9.859h16.2v.314l-7.032 8.917v5.149h6.279c4.835 0 7.033 1.821 7.033 5.84 0 3.076-1.884 4.897-4.773 4.897-3.328 0-5.149-1.821-5.651-5.839h-13.312c.753 10.047 7.409 15.384 18.963 15.384Z"/><path fill="#B90000" fill-rule="evenodd" d="m44.333 143.373 1.563-19.609 32.966 36.092-47.033 7.673 12.504-24.156Z" clip-rule="evenodd"/><path fill="#FF1D1D" fill-rule="evenodd" d="m0 138.684 2.984-79.147 44.76 27.282-1.848 36.945-1.563 19.609-12.504 24.156L0 138.684Z" clip-rule="evenodd"/><path fill="#CF0000" fill-rule="evenodd" d="m66.113 52.575-18.37 34.244-1.847 36.945 32.966 36.092 27.993-28.845-40.742-78.436Z" clip-rule="evenodd"/><path fill="#FF3232" fill-rule="evenodd" d="m68.49 14.21 2.273 29.697-4.65 8.668 40.742 78.436V52.575L68.489 14.209Z" clip-rule="evenodd"/><path fill="#FF6363" fill-rule="evenodd" d="M2.984 59.537 44.76 0l23.73 14.21 2.273 29.697-4.65 8.668-18.37 34.244L2.985 59.537Z" clip-rule="evenodd"/><path fill="#F5F5F5" d="M62.203 66.216H44.627v18.21h-18.21v17.576h18.21v18.21h17.576v-18.21h18.21V84.426h-18.21v-18.21Z"/></svg>
```

Comment:
What is the idea behind having the texts as `path` in `<svg>`?

Reply:
You mean "Helvetic Ruby"? That's how it got exported from Figma. It's because it's using a custom font. The payload is smaller when representing the letter shapes as SVG paths rather than embedding the entire font.